src/components/SliderProductItem.jsx

```diff
@@ -1,9 +1,13 @@
 const SliderProductItem = (props) => {
-    return (
+    return (!props.isDetail) ? (
```

Comment:
In my opinion something like looks better
```suggestion
    const SliderProductItem = ( {isDetail, id, url, alt, name }) => {
    return (
    <div key={props.item.id}>
            <img src={url} alt={alt} className="wz-slider-product-item-image"/>
            { isDetail && <h3 className="wz-center">{name}</h3> }  
     </div>
    )
}
```
Sorry if there is any syntax error